packages/grafbase-sdk/README.md

```diff
@@ -374,10 +383,10 @@ const github = connector.GraphQL({
 })
 ```
 
-Introspecting the connector namespace to the schema happens with the `introspect` method of the schema:
+Connectors can be added to the schema using `g.datasource()`, including an optional `namespace`:
 
 ```typescript
-g.datasource(contentful, { namespace: 'Contentful' })
+g.datasource(contentful)
```

Comment:
I think we should encourage namespace in all examples.

Reply:
Me too. I can revert but the line above explains the change, maybe I can be clearer. One shows without and one with otherwise we don't mention the optional namespace anywhere else.

Reply:
Got it. Perhaps we add a sentence about when optional namespaces are useful and the pitfalls of not using a namespace.

Reply:
I've added the `namspace` back in to both examples. The real documentation makes a note of the optional namespace and that is best practice to avoid collisions.